core/src/main/scala/kafka/server/AutoTopicCreationManager.scala

```diff
@@ -166,7 +177,28 @@ class DefaultAutoTopicCreationManager(
         debug(s"Auto topic creation completed for ${creatableTopics.keys}.")
         clearInflightRequests(creatableTopics)
       }
-    })
+    }
+
+    requestOpt match {
+      case Some(request) =>
+        val requestVersion =
+          channelManager.get.controllerApiVersions() match {
+            case None =>
+              ApiKeys.CREATE_TOPICS.latestVersion()
+            case Some(nodeApiVersions) =>
+              nodeApiVersions.latestUsableVersion(ApiKeys.CREATE_TOPICS)
+          }
+
+        val requestHeader = new RequestHeader(ApiKeys.CREATE_TOPICS,
+          requestVersion,
+          request.context.clientId(),
+          request.context.correlationId())
+        val envelopeRequest = ForwardingManager.buildEnvelopeRequest(request,
+          createTopicsRequest.build().serializeWithHeader(requestHeader))
+        channelManager.get.sendRequest(envelopeRequest, requestCompletionHandler)
```

Comment:
nit: we can probably avoid all of these `get` calls if we add something like this above:
```scala
val channelManager = this.channelManager.getOrElse {
  throw new IllegalStateException("Channel manager must be defined in order to send CreateTopic requests")
}
```